src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -1573,6 +1573,53 @@ const boost::optional<bool> CacheManager::LockScreenDismissalEnabledState()
   return empty;
 }
 
+const boost::optional<std::string>
+CacheManager::LockScreenDismissalWarningMessage(
```

Comment:
@IGapchuk  I feel like this function can be refactored little bit

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6

Reply:
This function was never refactored

Reply:
@jacobkeeler Thanks. This place has been refactored in 7f889b0543e993801001ae47e7375dbc3a238254, This commit is general feature PR - https://github.com/smartdevicelink/sdl_core/pull/2783